src/index.html

```diff
@@ -6,6 +6,37 @@
     <title>Wbpack Exercise</title>
 </head>
 <body>
-    <h1>Hello webpack!</h1>
+    <main>
+        <header>
+            <h1>Leaderboard</h1>
+        </header>
+        <section>
+            <div id="scores">
+                <div id="column1">
+                    <div id="heading">
+                        <h2>Recent scores</h2>
+                        <button class="btnstyle" id="refreshbtn">Refresh</button>
+                    </div>
+                    <div id="scorediv">
+                        <div class="score">Name 100</div>
+                        <div class="score" >Nmae 20</div>
+                        <div class="score">Name 50</div>
+                        <div class="score">Name 78</div>
+                        <div class="score">Name 125</div>
+                        <div class="score">Name 77</div>
+                        <div class="score">Name 42</div>
+                    </div>
+                </div>
+                <div id="column2">
+                    <h2>Add your score</h2>
+                    <form>
+                        <input type="text" id="name" name="fname" value="Your name" required ><br><br>
+                        <input type="text" id="score" name="lname" value="Your score" required ><br><br>
```

Comment:
## OPTIONAL
- [ ] It would be better to add the value of `your name & your score` as placeholders so users don't need to clear the value before inputting their name and scores 👍🏿